wp-seo-main.php

```diff
@@ -249,15 +249,21 @@ function _wpseo_deactivate() {
  * {@internal Unfortunately will fail if the plugin is in the must-use directory.
  *            {@link https://core.trac.wordpress.org/ticket/24205} }}
  *
- * @param int $blog_id Blog ID.
+ * @param int || object $blog The blog id or blog object, depending on the hook used to call this method.
```

Comment:
I think it would be clearer if this function would still receive a blog id. Maybe in case of a wp > 5.1 site, create a separate function that will call this function upon retrieving the blog_id from the blog object.